extensions/grafana-loki/tests/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+include(WholeArchive)
+
```

Comment:
This include no longer seems necessary

Reply:
Good point, updated in fc172520e6c3dbab68d5f9d37a76761927b02ab2